### ISSUE TITLE
Copyright waiver

### ISSUE THREAD

**Issue by FranzSkuffka** [mawww/kakoune#451]
I think the the copyright waiver handling is an unnecessary hurdle when contributing to this software. Would it not be enough to state in the contribution guidelines that you opt-in whenever you issue a pull request? Mentioning it in the context of PR's should be enough.

**Comment by mawww** [comment]
Hi,

The idea behind the copyright waiver commit is that it ends up in the git history, so we dont depend on data scattered around on github. It seems simpler to have every information in the git history directly.

**Comment by FranzSkuffka** [comment]
Ah I totally get where you are coming from. I have a different idea. When you issue your first pull request to the repository you add your name to the readme or a `contributors.md`. We still have a centralised solution and an accredition of contributors. Contributors automatically opt-in to the copyright waiver.

**Comment by FranzSkuffka** [comment]
This may be helpful.
https://github.com/MostlyAdequate/mostly-adequate-guide/blob/master/CONTRIBUTING.md#Translations

**Comment by mawww** [comment]
Hi,

Not sure how simpler it is, you only have to add the copyright waiver on your *first* contribution, once that is done, you are set. Ideally that waiver would be the first commit, but I am not very strict on that. It seems similar to have one commit touching contributors.md, or a commit creating the waiver.

The "creating a pull request implicitely accepts the licensing" is nice, but less explicit, its easy to say "I did not read CONTRIBUTING.md so I did not know", you cannot say you made a copyright waiver commit unintentionally.
